hare/eligibility/oracle.go

```diff
@@ -521,10 +504,6 @@ func (o *Oracle) IsIdentityActiveOnConsensusView(ctx context.Context, edID strin
 	}()
 	actives, err := o.actives(ctx, layer)
 	if err != nil {
-		if err == errGenesis { // we are in genesis
```

Comment:
Where is `errGenesis` defined? Is it still used anywhere? Do we still need it?

Reply:
Forgot to remove it, thanks for noticing - took care of it now.